
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -284,11 +284,7 @@ ENDIF()

 IF( EXIV2_ENABLE_PNG )
     IF( ZLIB_FOUND )
-        IF( MSVC )
-            TARGET_LINK_LIBRARIES( exiv2lib ${PRIVATE_VAR} optimized ${ZLIB_LIBRARIES} )
-        ELSE()
-            TARGET_LINK_LIBRARIES( exiv2lib ${PRIVATE_VAR} ${ZLIB_LIBRARIES} )
-        ENDIF()
+        TARGET_LINK_LIBRARIES( exiv2lib ${PRIVATE_VAR} ${ZLIB_LIBRARIES} )
     ENDIF()
 ENDIF()
